nodl_python/README.md

```diff
@@ -0,0 +1,2 @@
+# nodl_python: tools for parsing and analyzing nodl files
+![test nodl](https://github.com/ubuntu-robotics/nodl/workflows/test%20nodl/badge.svg?event=push)
```

Comment:
Let's move this badge up into the root README. No need for this package to have a README at this point, I don't think.